src/AutomatedCar/Keyboard.cs

```diff
@@ -1,12 +1,30 @@
 namespace AutomatedCar
 {
     using System.Collections.Generic;
+    using System.Linq;
     using Avalonia.Input;
 
     public static class Keyboard
     {
+        public static IReadOnlyCollection<Key> ToggleKeys = new List<Key>
+        {
+            Key.E,
+            Key.Q,
+            Key.A,
+            Key.D,
+            Key.R,
+        };
```

Comment:
Tetszik ez a megoldás, annyi csak hogy vannak a debugpacket ben nincsenek megcsinálva ugyanilyen toggle funkciók (Ultrahang: "u", Kamera: "Z", Radar: "I") ezeket beledobnád? 
Majd csinálok erre egy issuet mert az szerintem nem is lett erre.

Reply:
Persze beleírom azokat is.